app/forms/providers/gateway_evidence_form.rb

```diff
@@ -0,0 +1,147 @@
+module Providers
+  class GatewayEvidenceForm # rubocop:disable Metrics/ClassLength
+    include BaseForm
+
```

Comment:
A lot of this seems identical to the `StatementOfCaseForm`, in whcih case I think you could extract the common code out into a `BaseFileUploader` from, then derive the `StatementOfCaseForm` and this form from it.  It seems like more work now, but if ever we have to change it, it means a change in just one place.

Reply:
Yes great minds... I had written that as a ticket to do off the back of this. Thought I'd get this up and working with the right tests + functionality, and then look at refactoring the forms/uploader together later